README.md

```diff
@@ -72,9 +97,15 @@ To build Cuberite and the C++ Minecraft client:
 make
 ```
 
-### Run the Cuberite instance
+# Interacting with the Agent
+
+If you did a local install, run the following commands from the cloned repository directory.
+
+If you installed CraftAssist using Docker, run these commands in terminals inside of your Docker container (Docker has been configured to open shells in the correct directory). Also, Docker users should use the `python3` command instead of `python`.
```

Comment:
I suppose we have to run both cuberite and craftassist_agent in docker container terminal using python3. Could you please make this section more explicit and detailed by guiding the user to use the above command to open **TWO** docker container terminal and run with corresponding commands? Feel free to organize them in a way that you see fit. Thanks!